test/cljs/rems/testing.cljs

```diff
@@ -1,9 +1,18 @@
 (ns rems.testing
   (:require [re-frame.core :as rf]))
 
-(defn isolate-re-frame-state [f]
+(defn- clear-rf-state! []
+  #_(rf/clear-cofx)
+  #_(rf/clear-event)
+  #_(rf/clear-global-interceptor)
+  #_(rf/clear-sub)
+  (rf/clear-subscription-cache!))
+
+(defn init-client-fixture [f]
```

Comment:
Maybe this name could still be something else. I mean it talks about some client but I'm asking what client? Maybe `init-spa-fixture` or something related to that?